conn.go

```diff
@@ -76,7 +76,7 @@ func (conn *Conn) PassivePort() int {
 		return minPort + mrand.Intn(maxPort-minPort)
 	}
 	// let system automatically chose one port
-	return 0
+	return 20020 + mrand.Intn(200)
```

Comment:
If you're going to change the default, maybe we can update the README to include what it is.